<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qwdialogLocate</class>
 <widget class="QDialog" name="qwdialogLocate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>52</y>
     <width>30</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>列号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>133</y>
     <width>30</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>行号</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRow">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>52</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCol">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>133</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRow">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>53</y>
     <width>55</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>列增</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCol">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>134</y>
     <width>55</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>行增</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>52</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设定文字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>133</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>214</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设定文字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>214</y>
     <width>125</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
